src/components/support-form/steps/AdditionalQuestions.tsx

```diff
@@ -209,7 +212,7 @@ export default function AdditionalQuestions() {
   return (
     <Grid container spacing={6} justifyContent="center">
       <Grid item xs={12}>
-        <Typography variant="h4" align="center">
+        <Typography variant="h4" align="center" className={classes.subtitleText}>
```

Comment:
This way of handling it is okay, but the better way of handling it is with a custom Title component that has those styles internally.

With dedicated components, we embrace composition and single-purpose and we guarantee same result for all instances of the title.

Now the title can be blue but if we want to adjust the font size or underlying components we need to tweak 5+ files